ui/component/settingsStripeCard/view.jsx

```diff
@@ -224,6 +234,16 @@ const SettingsStripeCard = (props: Props) => {
             </FormField>
           </fieldset-section>
         </div>
+
+        <div className="stripe-billing-history">
+          <h2 className="stripe-billing-history__header">View billing history on Stripe</h2>
+          <Button
+            className="stripe-billing-history__button"
+            button="secondary"
+            label={__('Visit Stripe')}
+            navigate={`${STRIPE.STRIPE_BILLING_URL}?prefilled_email=${stripeBillingEmail}`}
```

Comment:
:small_red_triangle: Encode the prefill email per documentation

It didn't work for me because the email contained a plus (e.g. blah+test@gmail.com)

<img width="211" alt="image" src="https://user-images.githubusercontent.com/64950861/196755333-f8748c0b-8fa8-45f9-bcfb-24a8fb27913c.png">